Purchases/Public/RCPurchasesErrorUtils.m

```diff
@@ -62,6 +62,10 @@
             return @"There was an unknown backend error.";
         case RCReceiptInUseByOtherSubscriberError:
             return @"The receipt is in use by other subscriber.";
+        case RCInvalidAppleSubscriptionKeyError:
+            return @"Apple Subscription Key is invalid or it's not present. In order to provide subscription offers, you must first generate a subscription key, please see https://help.apple.com/app-store-connect/#/dev689c93225 for more info";
```

Comment:
"Apple Subscription Key is invalid or not present."

"In order to provide subscription offers, you must first generate a subscription key. Please see [link to our docs maybe?] for more info."

Reply:
I added a link to https://docs.revenuecat.com/docs. Don't want to use a specific link to a section of the docs since it will probably break in the near future.

Reply:
@rkotzy can probably provide you the url that will be available once we go live.